<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>675</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>675</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>675</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>TileMapper Lite</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressource/RessourceIcon.qrc">
    <normaloff>:/Assets/icon.png</normaloff>:/Assets/icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="windowFlags" stdset="0">
   <string>Qt::FramelessWindowHint</string>
  </property>
  <widget class="QWidget" name="view_main">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="view_right" native="true">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>401</width>
      <height>675</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="TilesetGraphicsView" name="graphicview_tileset">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>35</y>
       <width>334</width>
       <height>493</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
    </widget>
    <widget class="QLabel" name="la_tilechoose">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>530</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Tile</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicview_current_tile">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>560</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_layout_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>All Layout</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_layout_back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>560</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_layout_mid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Middleground</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_layout_front">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>620</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Foreground</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkbox_grid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>70</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Grid</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_mousey">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>650</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Row : 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="la_mousex">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>650</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Column : 0</string>
     </property>
    </widget>
   </widget>
   <widget class="MapGraphicsView" name="graphicview_map">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>942</width>
      <height>622</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="backgroundBrush">
     <brush brushstyle="NoBrush">
      <color alpha="255">
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </brush>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_new_map">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Ressource/RessourceIcon.qrc">
      <normaloff>:/Assets/new.png</normaloff>:/Assets/new.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_load_map">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>5</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Ressource/RessourceIcon.qrc">
      <normaloff>:/Assets/load_map_2.png</normaloff>:/Assets/load_map_2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_map">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>5</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Ressource/RessourceIcon.qrc">
      <normaloff>:/Assets/Save_3.png</normaloff>:/Assets/Save_3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_resize_map">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>5</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Ressource/RessourceIcon.qrc">
      <normaloff>:/Assets/Resize_2.png</normaloff>:/Assets/Resize_2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export_map">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>5</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Ressource/RessourceIcon.qrc">
      <normaloff>:/Assets/export.png</normaloff>:/Assets/export.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_import_tileset">
    <property name="geometry">
     <rect>
      <x>1250</x>
      <y>5</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../Ressource/RessourceIcon.qrc">
      <normaloff>:/Assets/load_tileset_2.png</normaloff>:/Assets/load_tileset_2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MapGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphics_views/MapGraphicsView.h</header>
  </customwidget>
  <customwidget>
   <class>TilesetGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphics_views/TilesetGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btn_new_map</tabstop>
  <tabstop>btn_load_map</tabstop>
  <tabstop>btn_save_map</tabstop>
  <tabstop>btn_resize_map</tabstop>
  <tabstop>btn_export_map</tabstop>
  <tabstop>btn_import_tileset</tabstop>
  <tabstop>rb_layout_all</tabstop>
  <tabstop>rb_layout_back</tabstop>
  <tabstop>rb_layout_mid</tabstop>
  <tabstop>rb_layout_front</tabstop>
  <tabstop>checkbox_grid</tabstop>
  <tabstop>graphicview_map</tabstop>
  <tabstop>graphicview_tileset</tabstop>
  <tabstop>graphicview_current_tile</tabstop>
 </tabstops>
 <resources>
  <include location="../Ressource/RessourceIcon.qrc"/>
 </resources>
 <connections/>
</ui>
